.kokoro

```diff
@@ -43,6 +43,7 @@ fix_bazel_imports() {
   rewrite_tests "s/import MaterialComponents.Material(.+)_(.+)/import components_\1_\2/"
   rewrite_tests "s/import MaterialComponents.Material(.+)Scheme/import components_schemes_\1_\1/"
   rewrite_tests "s/import MaterialComponents.MaterialMath/import components_private_Math_Math/"
+  rewrite_tests "s/import MaterialComponents.MaterialUIMetrics/import components_private_UIMetrics_UIMetrics/"
```

Comment:
should we get rid of all the MaterialUIMetrics in this PR?

Reply:
Moved to https://github.com/material-components/material-components-ios/pull/4294